plugins/utils/todo-comments.nix

```
@@ -371,17 +371,16 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin config {
   };
 
   extraOptions = {
-    keymaps =
-      let
-        mkKeymapOption =
-          optionName: funcName:
-          mkCompositeOption "Keymap settings for the `:${funcName}` function." {
-            key = mkOption {
-              type = types.str;
-              default = null;
-              description = "Key for the `${funcName}` function.";
-            };
+    keymaps = mapAttrs (
+      optionName: funcName:
+      mkNullOrOption' {
+        type = keymaps.mkMapOptionSubmodule {
+          defaults = {
+            mode = "n";
+            action = "<cmd>${funcName}<cr>";
```

```suggestion
            action = funcName;
```

Maybe rename the argument so we can do `inherit action`?